src/avt/Filters/avtResampleFilter.C

```diff
@@ -401,8 +407,27 @@ avtResampleFilter::ResampleInput(void)
     avtImagePartition partition(width, height, PAR_Size(), PAR_Rank());
     communicator.SetImagePartition(&partition);
     bool doDistributedResample = false;
+#if LIB_VERSION_LE(VTK,8,1,0)
 #ifdef PARALLEL
     doDistributedResample = atts.GetDistributedResample();
+#endif
+#else
+    bool doPerRankResample = false;
+#ifdef PARALLEL
+    // When running in parallel one can resample all of the data on to
```

Comment:
BTW...is there a doc update associated with this new rendering stuff? I thought I recall seeing @ARSanderson demo something about doc update too but I don't see it here if my recollection is correct (which it surely could not be). Maybe coming in another PR?

Reply:
There is an update to docs for Volume plot. Not part of a PR yet, not sure how best to reconcile the new version with the old, since it depends on VisIt being built with VTK 9.

I don't think the Resample filter changes have been documented.

Reply:
I believe that is correct as I cannot remember doing anything.